pyogrio/tests/test_geopandas_io.py

```diff
@@ -510,19 +653,21 @@ def test_write_mixed_geometries(tmp_path, driver, ext):
         crs="EPSG:4326",
```

Comment:
On the line above, can you add three additional geometries (multi-geometries)?

```
    MultiPoint([Point(1, 1), Point(2, 2)]),
    MultiLineString([LineString([(1, 1), (2, 2)]), LineString([(2, 2), (3, 3)])]),
    MultiPolygon([box(1, 1, 2, 2), box(2, 2, 3, 3)]),
```

that should catch the bug I mentioned above